esx_service/vmci/vmci_client.c

```diff
@@ -349,5 +366,22 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
 
+   ans->errBuf = malloc(512);
+   memset(ans->errBuf, 0, 512);
```

Comment:
just out of curiosity - if you allocate and then null it, why not use calloc (I do not think we need to be allocating anyways).